playbooks/demo_apache-server.yml

```diff
@@ -14,16 +16,9 @@
       download_dir: "~"
       target_dir: "/tmp/.ansible.cpdir"
 
-
-- name: "Install the Apache HTTP server using yum"
-  hosts: apache
-  gather_facts: no
-  collections:
-  - ibm.power_aix
-
   tasks:
```

Comment:
do you need two `tasks` section?

Reply:
for some reason, the tasks after the role don't get called if there's not another tasks section.

Reply:
that's strange.